src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1293,7 +1430,19 @@ mod tests {
             hydraulics: A320Hydraulic,
             overhead: A320HydraulicOverheadPanel,
             engine_fire_overhead: A320EngineFireOverheadPanel,
+            emergency_electrical_overhead: A320EmergencyElectricalOverheadPanel,
             landing_gear: LandingGear,
+
+            // Electric buses states to be able to kill them dynamically
```

Comment:
**thought**: I think the field names are clear enough (though I would suggest changing `alive` to `powered` to be in line with all the other naming) and don't require a comment.